server/src/domain/asset/response-dto/exif-response.dto.ts

```diff
@@ -24,6 +24,7 @@ export class ExifResponseDto {
   state?: string | null = null;
   country?: string | null = null;
   description?: string | null = null;
+  projectionType?: string | null = null;
```

Comment:
Why not `ProjectionType`?

Reply:
I tried that, this is exif-response and it works only when string is set, not ProjectionType